tests/test.go

```diff
@@ -0,0 +1,153 @@
+package tests
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"reflect"
+
+	"github.com/PPSKSY-Cluster/backend/api"
+	"github.com/PPSKSY-Cluster/backend/db"
+	"github.com/gofiber/fiber/v2"
+	"github.com/joho/godotenv"
+	"github.com/stretchr/testify/assert"
+)
+
+const TIMEOUT int = -1
+
+func setupTestApplication() (*fiber.App, error) {
+	// Server setup : load env file, setup db, setup router
+	if err := godotenv.Load("./.env"); err != nil {
```

Comment:
Do we plan to use different env files for testing and running? If so this should be stated in the README

Reply:
Ah yes it seems like I accidentaly commited that [here](https://github.com/PPSKSY-Cluster/backend/pull/28/commits/16a42874968227ed1a693c37401fdcfc7c80653f). I will try to clean that up. The main reason is that tests tend to clutter databases, so one would probably want to use a different one for testing, that can be dropped after each test (what testInterface.go is for).